x-pack/plugins/security/server/config.ts

```diff
@@ -60,42 +174,96 @@ export const ConfigSchema = schema.object({
   }),
 });
 
-export function createConfig$(context: PluginInitializerContext, isTLSEnabled: boolean) {
-  return context.config.create<TypeOf<typeof ConfigSchema>>().pipe(
-    map(config => {
-      const logger = context.logger.get('config');
+export function createConfig(
```

Comment:
__note:__ there was no strong reason to use Observables here and now it's much easier to use in tests when we need to test config permutations (and we have a lot of them).